tests/test_invenio_i18n_selectors.py

```diff
@@ -156,3 +156,42 @@ def test_get_locale_anonymous_user(app):
 
     with app.test_request_context():
         assert "en" == get_locale()
+
+
+def test_get_locale_forced_locale(app):
+    """Test retrieving locale via forced locale."""
+    from invenio_i18n import force_locale
```

Comment:
minor: is it necessary to place the import statement here? Could you move that to the beginning of the file?